subsys/zigbee/lib/zigbee_app_utils/zigbee_app_utils.c

```diff
@@ -856,3 +868,47 @@ void zigbee_configure_sleepy_behavior(bool enable)
 	}
 }
 #endif
+
+/* Wrapper allowing to schedule zb_bdb_reset_via_local_action as an alarm */
+static void factory_reset(zb_uint8_t arg)
+{
+	ZVUNUSED(arg);
+	LOG_DBG("Perform factory reset");
+	zb_bdb_reset_via_local_action(0);
```

Comment:
What's the parameter of the `zb_bdb_reset_via_local_action`?

Reply:
@tomchy From description of the function zb_bdb_reset_via_local_action: "buffer reference (if 0, buffer will be allocated automatically)"